src/routes/discussions/[id]/index.tsx

```diff
@@ -20,6 +23,17 @@ export const useFormLoader = routeLoader$<InitialValues<ReplyFormType>>(() => ({
   comment: "",
 }));
 
+export const useToggleReaction = routeAction$(async (data) => {
```

Comment:
Could be refactored into a `server$` action to access the user's `cookie.oauth` and retrieve their stored authentication token.

```typescript
server$(async function(data) {
  ...
  const oauthCookie = this.cookies.get('oauth');
  ...
```

This would allow you to call `queryUserGraphQl` which queries as the user instead of the GitHub App